packages/labs/eleventy-plugin-lit/test/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "__note__": "Switches Node into module mode for tests",
+  "type": "module"
+}
```

Comment:
If we need this just for tests, what do you think about adding `NODE_OPTIONS='--input-type=module'` in front of the test command to achieve that? We can also use [cross-env](https://www.npmjs.com/package/cross-env) for cross-platform safe way of setting environment variables if wanted.

Reply:
The `"type":"module"` approach seems simpler to me, and it means `npx uvu` will work normally, instead of having to remember to deal with extra envs.